scripts/gitlab/test-cpp.sh

```diff
@@ -2,16 +2,15 @@
 echo "________Running test-cpp.sh________"
 set -e # fail on any error
 set -u # treat unset variables as error
+set -B # enable brace expansion
 #use nproc `linux only
 THREADS=$(nproc)
-export CC="sccache gcc"
-export CXX="sccache g++"
 
 echo "________Running the C++ example________"
 DIR=parity-clib/examples/cpp/build
 mkdir -p $DIR
 cd $DIR
-cmake ..
+cmake -DCMAKE_C{,XX}_COMPILER_LAUNCHER=sccache
```

Comment:
Seems to be executed from the wrong directory

```suggestion
cmake .. -DCMAKE_C{,XX}_COMPILER_LAUNCHER=sccache
```